.github/workflows/ci.yml

```diff
@@ -337,9 +337,85 @@ jobs:
           cargo test --package libparsec_crypto --features use-rustcrypto
           cargo test --workspace --features mock-time --exclude libparsec_crypto
 
+  ##############################################################################
+  #                            🐜 Wasm tests                                   #
+  ##############################################################################
+
+  test-wasm:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - crate_to_test:
```

Comment:
Why a matrix with only a single element ?

Reply:
I will refactor that

Reply:
I need a matrix with 1 element because I need something that is able to hold a `String[]` to configure the task `Check changed files`.
I've tried to set this value in `env` but It does not accept a list :(

Reply:
Add a comment to explain it then ;-)

Reply:
👌